.circleci/config.yml

```diff
@@ -84,52 +85,57 @@ defaults: &defaults
 
 version: 2
 jobs:
-  test-node-8:
+  test-javascript-node-8:
     <<: *defaults
     docker:
       - image: circleci/node:8
     steps:
       - checkout
-      - run: npm install --no-package-lock
+      - run: 
+          name: Install Dependencies
+          command: |
+            npm config set spin=false
+            npm config set progress=false
+            npm install --no-package-lock
       - run: |
           npm test -- --maxWorkers=2
           npm run lint
           npm run flow -- check
-# eslint - doesn't run on non-PR builds
-      - run:
-          name: Analyze Code
-          command: |
-            if [ -n "$CIRCLE_PR_NUMBER" ]; then
-              npm install github@0.2.4
-              cat <(echo eslint; npm run lint --silent -- --format=json; echo flow; npm run flow --silent -- check --json) | GITHUB_TOKEN="af6ef0d15709bc91d""06a6217a5a826a226fb57b7" CI_USER=$CIRCLE_PROJECT_USERNAME CI_REPO=$CIRCLE_PROJECT_REPONAME PULL_REQUEST_NUMBER=$CIRCLE_PR_NUMBER node bots/code-analysis-bot.js
-            else 
-              echo "Skipping code analysis."
-            fi
 
-  test-node-6:
+  test-javascript-node-6:
```

Comment:
`js` is more concise?

Reply:
Sure. Feel free to edit the file on the PR (if you have access)

Reply:
Ah, true, will play around :)